src/main/clojure/starfish/core.cljc

```diff
@@ -318,11 +318,11 @@
   (^Asset [^Operation operation params]
    (let [job (invoke operation params)
          resp (.getResult job)]
-     resp))
+     (into {} resp)))
```

Comment:
Probably want a keywordize-keys anywhere that we return a map from Java into Clojure land?